invenio_app_rdm/config.py

```diff
@@ -539,6 +539,12 @@ def files_rest_permission_factory(obj, action):
 SEARCH_UI_SEARCH_TEMPLATE = "invenio_app_rdm/records/search.html"
 """Search page's base template."""
 
+DEPOSIT_CREATE_TEMPLATE = "invenio_app_rdm/records/deposit.html"
```

Comment:
Just to align with the rest of the [deposit config](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/config.py#L607) I would prepend the `APP_RDM` into the template name.

Reply:
what would be the use case for bringing this change, since the templates can be overridden by placing them in correct folder structure?

Reply:
@kpsherva Please read [discord ](https://discord.com/channels/692989811736182844/704625518552547329/1047339417322405918)